packages/js/components/src/sortable-list/README.md

```diff
@@ -0,0 +1,27 @@
+SortableList
+===
+
+This component provides a wrapper to allow dragging and sorting of items.
+
+## Usage
+
+This component accepts any valid JSX elements as children.  Adding a `key` to elements will provide a way to later identify the order of these elements in callbacks.
+
+```jsx
+<SortableList onOrderChange={ ( items ) => console.log( 'Items have been reordered:', items ) }>
+    <div key="item-1">List item 1</div>
+    <div key="item-2">List item 2</div>
+    <div key="item-3">List item 3</div>
+</SortableList>
+```
+
+### Props
+
+Name | Type | Default | Description
+--- | --- | --- | ---
+`children` | JSX.Element | JSX.Element[] | `undefined` | The draggable items in the list
```

Comment:
Your pipe between types is pushing your description out of the table. Recommend replacing with an entity or escaping with a backslash.